[2025-07-03T03:12:31.824+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-03T03:12:31.853+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-07-02T03:00:00+00:00 [queued]>
[2025-07-03T03:12:31.865+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-07-02T03:00:00+00:00 [queued]>
[2025-07-03T03:12:31.866+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-03T03:12:31.880+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-07-02 03:00:00+00:00
[2025-07-03T03:12:31.889+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=7587) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-03T03:12:31.892+0000] {standard_task_runner.py:72} INFO - Started process 7588 to run task
[2025-07-03T03:12:31.891+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup_resources', 'scheduled__2025-07-02T03:00:00+00:00', '--job-id', '318', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpi8y4yn8n']
[2025-07-03T03:12:31.892+0000] {standard_task_runner.py:105} INFO - Job 318: Subtask cleanup_resources
[2025-07-03T03:12:31.959+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-07-02T03:00:00+00:00 [running]> on host 74e14c8611e5
[2025-07-03T03:12:32.053+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jasjot Parmar' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-07-02T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-02T03:00:00+00:00'
[2025-07-03T03:12:32.055+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-03T03:12:32.078+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-03T03:12:32.080+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-07-03T03:12:32.098+0000] {subprocess.py:99} INFO - Output:
[2025-07-03T03:12:32.109+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-03T03:12:32.140+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-03T03:12:32.141+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup_resources, run_id=scheduled__2025-07-02T03:00:00+00:00, execution_date=20250702T030000, start_date=20250703T031231, end_date=20250703T031232
[2025-07-03T03:12:32.193+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-03T03:12:32.209+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-03T03:12:32.211+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-07-04T01:20:42.350+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-04T01:20:42.379+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-07-02T03:00:00+00:00 [queued]>
[2025-07-04T01:20:42.389+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-07-02T03:00:00+00:00 [queued]>
[2025-07-04T01:20:42.390+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-04T01:20:42.402+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup_resources> on 2025-07-02 03:00:00+00:00
[2025-07-04T01:20:42.411+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=247) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-04T01:20:42.413+0000] {standard_task_runner.py:72} INFO - Started process 248 to run task
[2025-07-04T01:20:42.413+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup_resources', 'scheduled__2025-07-02T03:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmp8re05c8r']
[2025-07-04T01:20:42.414+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask cleanup_resources
[2025-07-04T01:20:42.458+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup_resources scheduled__2025-07-02T03:00:00+00:00 [running]> on host 8862b69bc4b5
[2025-07-04T01:20:42.542+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Jasjot Parmar' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup_resources' AIRFLOW_CTX_EXECUTION_DATE='2025-07-02T03:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-02T03:00:00+00:00'
[2025-07-04T01:20:42.544+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-04T01:20:42.564+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-04T01:20:42.566+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'rm -f /app/tmp/*.pkl']
[2025-07-04T01:20:42.581+0000] {subprocess.py:99} INFO - Output:
[2025-07-04T01:20:42.589+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-04T01:20:42.614+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-04T01:20:42.615+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup_resources, run_id=scheduled__2025-07-02T03:00:00+00:00, execution_date=20250702T030000, start_date=20250704T012042, end_date=20250704T012042
[2025-07-04T01:20:42.670+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-04T01:20:42.691+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-04T01:20:42.693+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
